src/pages/messages/components/NviCandidateListItem.tsx

```diff
@@ -33,8 +34,9 @@ export const NviCandidateListItem = ({ nviCandidate, offsetNextCandidate }: NviC
 
   const myApproval = nviCandidate.approvals.find((approval) => approval.id === user?.topOrgCristinId);
 
-  const offsetNextCandidateState: CandidateOffsetState = {
-    offsetNextCandidate: offsetNextCandidate,
+  const candidateOffsetState: CandidateOffsetState = {
+    currentOffset,
+    nviQuery: nviListQuery,
```

Comment:
rename hele veien opp når vi først er i gang?
```suggestion
    nviQuery,
```